src/pages/TaskRun/TaskRun.vue

```diff
@@ -115,6 +119,45 @@ export default {
     },
     parseMarkdown(md) {
       return parser(md)
+    },
+    async saveTaskRunName(e) {
+      const previousName = this.taskRun.name
+      if (previousName === e) return
+
+      try {
+        this.taskRunNameLoading = true
+
+        const { data } = await this.$apollo.mutate({
+          mutation: require('@/graphql/Mutations/set-task-run-name.gql'),
+          variables: {
+            input: {
+              task_run_id: this.taskRunId,
+              name: e
+            }
+          }
+        })
+
+        await this.$apollo.queries.taskRun.refetch()
+
+        if (!data.set_task_run_name.success) {
+          throw new Error(data.set_task_run_name.error)
+        }
+
+        this.setAlert({
+          alertShow: true,
+          alertMessage: `<span class="font-weight-medium">${previousName}</span> has been renamed to <span class="font-weight-medium">${e}</span>`,
```

Comment:
Since not all task runs have names, previousName can come up as null, which looks a bit odd.  Could we add something like this:
```suggestion
          alertMessage: `<span class="font-weight-medium">${previousName || 'Your task run'}</span> has been renamed to <span class="font-weight-medium">${e}</span>`,
```